VideoLocker/src/main/java/org/edx/mobile/module/notification/ParseSyncTask.java

```diff
@@ -20,16 +20,12 @@ public ParseSyncTask(Context context) {
 
     @Override
     public Void call( ) throws Exception{
-        try {
-            subscribedChannels = ParseInstallation.getCurrentInstallation().getList("channels");
-        } catch (Exception ex) {
-            handle(ex);
-            logger.error(ex);
-        }
+        subscribedChannels = ParseInstallation.getCurrentInstallation().getList("channels");
         return null;
     }
 
     public void onException(Exception ex){
-          //do nothing.?
+        super.onException(ex);
+        //do nothing.?
```

Comment:
Nothing has really been changed in the implementation here. Previously, the
exception was caught and handled by the `call()` method itself, now it's
done by the base implementation of the `onException()` callback, and the
no-op override of it was superfluous in both the previous and current
implementations. I tried not to make additional fixes here given the large
amount of change I was dealing with (so that it's scope was exactly defined
and easily checked), but we can simply remove the method override in this
case I guess.

On Mon, Mar 21, 2016 at 6:34 PM, Christopher Lee <notifications@github.com>
wrote:

> In
> VideoLocker/src/main/java/org/edx/mobile/module/notification/ParseSyncTask.java
> <https://github.com/edx/edx-app-android/pull/627#discussion_r56821622>:
>
> >          return null;
> >      }
> >
> >      public void onException(Exception ex){
> > -          //do nothing.?
> > +        super.onException(ex);
> > +        //do nothing.?
>
> Should this comment be removed now?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-app-android/pull/627/files/0e76ea287f00537ec5a4147394b1415ad51a630f#r56821622>
>